standard/structs.md

```diff
@@ -502,6 +505,50 @@ Automatically implemented properties ([§15.7.4](classes.md#1574-automatically-i
 
 > *Note*: This access restriction means that constructors in structs containing automatically implemented properties often need an explicit constructor initializer where they would not otherwise need one, to satisfy the requirement of all fields being definitely assigned before any function member is invoked or the constructor returns. *end note*
 
+A *property_declaration* ([§14.7.1](classes.md#1471-general)) for an instance property in a *struct_declaration* may contain the *property_modifier* `readonly`. However, a static property shall not contain that modifier.
+
+It is a compile-time error to attempt to modify the state of an instance struct variable via a readonly property declared in that struct.
+
+It is a compile-time error for an automatically implemented property having a `readonly` modifier, to also have a `set` accessor.
+
+It is a compile-time error for an automatically implemented property in a `readonly` struct to have a `set` accessor.
+
+An automatically implemented property declared inside a `readonly` struct need not have a `readonly` modifier, as its `get` accessor is implicitly assumed to be readonly.
+
+It is a compile-time error for a property to have a `readonly` modifier on both the `get` and the `set` accessors.
+
+It is a compile-time error to have a `readonly` modifier on a property itself as well as on either of its `get` and `set` accessors.
+
+If the `get` accessor has a `readonly` modifier, the `set` shall exist and shall not have that modifier.
```

Comment:
Agree, assuming CS8664 is the intended semantics and not a Roslyn quirk.

IIRC conformance allows *additional* members other than `get` & `set` so “all of its accessors” would be invalid in such a case. I'll offer the following:

```suggestion
If the `get` or `set` accessor has a `readonly` modifier then a sibling `set` or `get` accessor shall exist and shall not have that modifier.
```

Could use “corresponding” rather than “sibling”